src/main/java/net/gecore/streamerendpoints/service/igdb/IGDBGameService.java

```diff
@@ -0,0 +1,41 @@
+package net.gecore.streamerendpoints.service.igdb;
+
+import net.gecore.streamerendpoints.configuration.TwitchConfig;
+import net.gecore.streamerendpoints.service.twitch.AuthService;
+import net.gecore.streamerendpoints.service.twitch.TwitchAPIException;
+
+import net.gecore.streamerendpoints.service.utils.JsonPathUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class IGDBGameService {
+    private final IGDBAPI igdbapi;
+    private final TwitchConfig twitchConfig;
+    private final AuthService authService;
+//    private final Logger LOGGER = LoggerFactory.getLogger(IGDBService.class);
```

Comment:
It's been commented out because I'm unsure how you want me to use the Logger library. For example, you've set the logger in `TwitchAPI.java` to `TwitchAPI.class`. However, in `TwitchGameService.java` you've set it to `StreamService.class`. How would you like this used in IGDBGameService?

Reply:
True, that's definitely a copy paste mistake just like here. Still remove it for now, there's almost never a reason to commit commented code especially in a PR. It's just a bad practice. _Might be_ is a good way to have a lot of lines doing nothing.

Logging makes more sense once the entire flow is ready.